rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java

```diff
@@ -157,5 +157,9 @@ public BridgeDevNetConstants(List<BtcECKey> federationPublicKeys) {
         maxDepthBlockchainAccepted = 25;
 
         minimumPegoutValuePercentageToReceiveAfterFee = 20;
+
+        utxoExpirationCheckpointRskBlockHeight = 40000; // rounded 2 weeks of RSK blocks
```

Comment:
```suggestion
        utxoExpirationCheckpointRskBlockHeight = 40_000; // rounded 2 weeks of RSK blocks
```
Makes it clearer, some in other files

Reply:
Resolved